### ISSUE TITLE
Sidenav notes button in reader tab does not display notes list on Space

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#5306]
https://forums.zotero.org/discussion/124326/cannot-open-notes-pane-from-keyboard

When the notes button in sidenav of the reader tab is activated with Space, focus just moved to the context pane without displaying the list of notes.

This was initially implemented in https://github.com/zotero/zotero/pull/4235.

**Comment by abaevbog** [comment]
Expected:

https://github.com/user-attachments/assets/271eb038-511d-4e12-8dd4-8373ca370fd1

Current:

https://github.com/user-attachments/assets/a5296c19-f03f-4b6b-b33d-f137a369725e
